@navikt/core/react/src/overlays/action-menu/ActionMenu.stories.tsx

```diff
@@ -0,0 +1,852 @@
+      setCheckedItems((prevState) => ({
+        ...prevState,
+        [checkboxId]: !prevState[checkboxId],
+      }));
+    };
+
+    return (
+      <ActionMenu open={props.open}>
+        <ActionMenu.Trigger>
+          <button>Open action</button>
+        </ActionMenu.Trigger>
+        <ActionMenu.Content>
+          <ActionMenu.CheckboxItem
+            checked={
+              Object.values(checkedItems).every(Boolean)
+                ? true
+                : Object.values(checkedItems).some(Boolean)
+                  ? "indeterminate"
+                  : false
+            }
+            onCheckedChange={() =>
+              Object.values(checkedItems).every(Boolean)
+                ? checkedItems
+                : setCheckedItems((prevState) =>
+                    Object.keys(prevState).reduce(
+                      (acc, key) => ({ ...acc, [key]: true }),
+                      {} as typeof checkedItems,
+                    ),
+                  )
+            }
+          >
+            Select all
+          </ActionMenu.CheckboxItem>
+          <ActionMenu.Group label="Group 1">
+            <ActionMenu.CheckboxItem
+              checked={checkedItems.checkbox1}
+              onCheckedChange={() => handleCheckboxChange("checkbox1")}
+            >
+              Checkbox 1
+            </ActionMenu.CheckboxItem>
+            <ActionMenu.CheckboxItem
+              checked={checkedItems.checkbox2}
+              onCheckedChange={() => handleCheckboxChange("checkbox2")}
+            >
+              Checkbox 1
+            </ActionMenu.CheckboxItem>
+          </ActionMenu.Group>
+          <ActionMenu.Group label="Group 2">
+            <ActionMenu.CheckboxItem
+              checked={checkedItems.checkbox3}
+              onCheckedChange={() => handleCheckboxChange("checkbox3")}
+            >
+              Checkbox 3
+            </ActionMenu.CheckboxItem>
+            <ActionMenu.CheckboxItem
+              checked={checkedItems.checkbox4}
+              onCheckedChange={() => handleCheckboxChange("checkbox4")}
+            >
+              Checkbox 4
+            </ActionMenu.CheckboxItem>
+          </ActionMenu.Group>
+        </ActionMenu.Content>
+      </ActionMenu>
+    );
+  },
+  decorators: [DemoDecorator],
+};
+
+export const RadioGroups: Story = {
+  render: (props) => {
+    const [group1Value, setGroup1Value] = useState("1");
+    const [group2Value, setGroup2Value] = useState("4");
+
+    // Step 3: Handle value change
+    const handleGroup1ValueChange = (value: string) => {
+      setGroup1Value(value);
+    };
+
+    const handleGroup2ValueChange = (value: string) => {
+      setGroup2Value(value);
+    };
+
+    return (
+      <ActionMenu open={props.open}>
+        <ActionMenu.Trigger>
+          <button>Open action</button>
+        </ActionMenu.Trigger>
+        <ActionMenu.Content>
+          <ActionMenu.RadioGroup
+            onValueChange={handleGroup1ValueChange}
```

Comment:
```suggestion
            onValueChange={setGroup1Value}
```
and similarly for the other one

Reply:
Fixed 👍